### DISCUSSION TITLE
"Task lists!" give feedback (unclear on what)

### DISCUSSION THREAD

**Discussion by peppy** [community/community#5211] [Projects and Issues]
We are seeing this recently on our issues pages:

![20210820 161443 (Safari)](https://user-images.githubusercontent.com/191335/130194997-59468789-c2fe-40cc-907d-5ad07785f23b.png)

But no one I have asked knows what this means, or what to give feedback on. It links to https://github.com/github/feedback/discussions/categories/issues-feedback which doesn't really answer any questions, either. If this is related to markdown lists, then it's something we've been using for years now and have not noticed any new features (unless it's related to the full PR title now displaying inline).

If it's meant to be something else that should be apparent enough to not require explanation, then it feels like the rollout has not actually been done for our repository, while the message for feedback is showing.

**Comment by VigneshPT** [comment]
I believe the `[ ]` type or the checkbox list is now updated and being called the `Task Lists`.. I can see that now we are able to re-order the tasks by drag and drop. Also, a button for "Convert to Issue" shows when hovering over the Task item.

**Reply by pmcelhaney** [reply]
The drag and drop is not new but "Convert to Issue" is. I just tried clicking the button on a random task to see how it works. It spun for a while. I refreshed the page and now both the "convert to issue" button and "Tasks Lists! Give feedback" have disappeared. 🙃
